src/lua/stack/type.cr

```diff
@@ -34,5 +34,41 @@ module Lua
     def typename(type : TYPE)
       String.new LibLua.typename(@state, type.value)
     end
+
+    def crystal_type_at(pos : Int)
```

Comment:
Would you mind to add some method docs with examples? This is a public API and would be nice to have some hints